docs/source/tracking/artifacts-stores.rst

```diff
@@ -0,0 +1,200 @@
+===============
+Artifact Stores
+===============
+
+The artifact store is a core component in `MLflow Tracking <../index.html>`_ where MLflow stores (typicaly large) arifacts
+for each run such as model weights (e.g. a pickled scikit-learn model), images (e.g. PNGs), model and data files (e.g. `Parquet <https://parquet.apache.org/>`_ file). 
+Note that metadata like parameters, metrics, and tags are stored in `backend store <backend-stores.html>`_, the other component of the MLflow Tracking.
```

Comment:
```suggestion
Note that metadata like parameters, metrics, and tags are stored in a `backend store <backend-stores.html>`_ (e.g., PostGres, MySQL, or MSSQL Database), the other component of the MLflow Tracking.
```
I think listing out examples of the backend stores might be helpful, just like you did with the artifact store examples :)